components/ClassDeduction/ClassDeduction.module.css

```
@@ -1,24 +1,33 @@
+  background-color: #2a2a2a;
 }
 
-.deduction:hover {
-  border-color: #007bff;
-  box-shadow: 0 6px 20px rgba(0, 123, 255, 0.15);
-  transform: translateY(-1px);
+.deductionContainer {
+  margin-left: 10px;
+  display: block;
+  background-color: white;
+  margin: 7px 0px;
```

Stylelint (17.13.0)

[error] Overridden property "margin-left" by shorthand "margin" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)